powerquery-docs/query-folding-basics.md

```diff
@@ -66,7 +66,7 @@ The following diagram explores the process that occurs when a query is evaluated
 >[!NOTE]
 >While this example showcases a query with a SQL Database as a data source, the concept applies to queries with or without a data source.
 
-When Power Query reads your M script, it runs the script through an optimization process to more efficiently evaluate your query. In this process, it determines which steps (transforms) from your query can be offloaded to your data source. It also determines which other steps need to be evaluated using the Power Query engine.
+When Power Query reads your M script, it runs the script through an optimization process to evaluate your query. In this process, it determines which steps (transforms) from your query can be offloaded to your data source. It also determines which other steps need to be evaluated using the Power Query engine.
```

Comment:
```suggestion
When Power Query reads your M script, it runs the script through an optimization process to evaluate your query. In this process, it determines which steps (transforms) from your query can be offloaded to your data source.
```

this does not add any value because "other steps" is unclear. isn't it sufficient to have the first sentence "runs the script through an optimization process to evaluate your query"?